src/CodeGen_Hexagon.cpp

```diff
@@ -577,19 +577,23 @@ void CodeGen_Hexagon::push_buffer(const std::string &name, int dimensions, llvm:
         // extern stages on hexagon that take buffers passed through from
```

Comment:
Ron, see the addon_lowlevel_code branch. Look at ExternalCode and the new append method in Halide::Module.